primefaces/src/main/resources/META-INF/resources/primefaces/forms/forms.css

```diff
@@ -829,6 +829,11 @@ div.ui-button, .ui-splitbutton {
     position: absolute;
 }
 
+.ui-password::-ms-reveal,
```

Comment:
I think we should add a class if `toggleMask="true"` and only act if that class is present.